src/main/java/com/google/codeu/data/User.java

```diff
@@ -1,21 +1,77 @@
 package com.google.codeu.data;
 
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Query.Filter;
+import com.google.appengine.api.datastore.Query.FilterOperator;
+import com.google.appengine.api.datastore.Query.FilterPredicate;
+import com.google.codeu.data.Datastore;
+import com.google.codeu.data.User;
+import java.util.ArrayList;
+import java.util.List;
+
 public class User {
 
-    private String email;
-    private String aboutMe;
+  private String email;
+  private String aboutMe;
+  private transient String id;
```

Comment:
i looked up what transient was in java, but i'm curious about the use case here and whether we need to add it to other files?

Reply:
Transient means that Gson won't serialize it when sending it down to the client. I don't know whether exposing user IDs is sensitive or not, so I'm choosing not to send it down. I don't think there are other places where we need to add `transient`.